workspaces/optic-engine/src/queries/spectacle/spec_choices.rs

```diff
@@ -98,7 +98,7 @@ impl<'a> ShapeChoiceQueries<'a> {
       .into_iter()
       .map(move |choice| match choice.core_shape_kind {
         ShapeKind::ObjectKind => {
-          let object_fields = queries.resolve_shape_field_id_and_names(&choice.shape_id);
+          let object_fields = queries.resolve_shape_field_id_and_names(&choice.shape_id, true);
```

Comment:
In the external / spectacle projection / shapeViewerProjection - we want to include removed nodes, so we can display the data associated with the removed field. (TODO test this out)